adig.c

```diff
@@ -825,3 +831,26 @@ static void append_addr_list(struct ares_addr_node **head,
   else
     *head = node;
 }
+
+
+/* Information from the man page. Formatting taken from man -h */
+static void print_help_info_adig() {
+    printf("adig, version %s \n\n", ARES_VERSION_STR);
+    printf("usage: adig [-h] [-d] [-f flag] [-s server] [-c class] [-t type] [-T|U port] name ...\n\n"
+    "  d : Print some extra debugging output.\n"
```

Comment:
Why ` d ` and not ` -d ...` as is more common in help-screens?
(same in all 3 `a*.c` files).